app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,45 @@
+class QuestionsController < ApplicationController
+  before_action :find_test, only: %i[index create]
+  before_action :find_question, only: %i[show destroy]
+
+  rescue_from ActiveRecord::RecordNotFound, with: :rescue_with_question_not_found
+
+  def index
+    render plain: @test.questions.pluck(:text)
+  end
+
+  def new
+  end
+
+  def create
+    @question = @test.questions.create!(question_params)
+    render plain: "Question was successfully created!"
+  end
+
+  def show
+    render plain: @question.text
+  end
+
+  def destroy
+    @question.destroy
+    render plain: "Question has been deleted!"
+  end
+
+  private
+
+  def find_question
+    @question = Question.find(params[:id].to_i)
```

Comment:
```suggestion
    @question = Question.find(params[:id])
```
`ActiveRecord` работает и с числами и со строками, явно переводить в число нет необходимости